xbmc/utils/SeekHandler.cpp

```diff
@@ -141,7 +141,10 @@ void CSeekHandler::Seek(bool forward, float amount, float duration /* = 0 */, bo
       if (g_infoManager.GetTotalPlayTime())
         percentPerSecond = 100.0f / (float)g_infoManager.GetTotalPlayTime();
 
-      m_percent = m_percentPlayTime + percentPerSecond * GetSeekSeconds(forward);
+      int seekSeconds = GetSeekSeconds(forward);
+      g_infoManager.SetSeekStepSize(seekSeconds);
```

Comment:
it makes no different IMO as Player.SeekOffset is displayed after seek is performed and Player.SeekStepSize becomes empty.